thirdparty/openapi3/schemas/VerifyThirdPartyAuthorizationRequest.yaml

```diff
@@ -0,0 +1,32 @@
+title: VerifyThirdPartyAuthorizationRequest
```

Comment:
Should this be renamed to ThirdPartyAuthorization**IDPost**Request to fall in line with other Mojaloop fspiop definitions?

Reply:
Yeah probably. My only thought is that `VerifyThirdPartyAuthorizationRequest` is a lot more descriptive

Reply:
we need some convention